scripts/run_pattern_check.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Pre-submission script for Oppia.
+# This script checks for unaccepted text patterns in the commit.
+
+import subprocess
+
+BAD_PATTERNS = {'__author__': 'Please remove author tags from \
+this file.',
+                'datetime.datetime.now()': 'Please use datetime.datetime.u\
+tcnow() instead of datetime.datetime.now().'
+               }
+
+def _get_changed_filenames():
```

Comment:
Yes, some changes are needed for testing all the files.
It can be used on both the places, locally and Travis. We can use it individually, but combining it with any of the scripts will make it easier for user to perform the test.